modules/pytket-pyzx/docs/intro.txt

```diff
@@ -15,7 +15,9 @@ PyZX representations.
 ``pytket-pyzx`` is available for Python 3.8, 3.9 and 3.10, on Linux, MacOS and
 Windows. To install, run:
 
-``pip install pytket-pyzx``
+::
```

Comment:
is this change intended in all intro files?

Reply:
Yes I intended to change

``pip install pytket-X``
to
::

    pip install pytket-X

Reply:
In all intro files* 

This puts the installation command in a box which I think looks nicer.

Reply:
Great